dev/ComboBox/ComboBox_themeresources.xaml

```diff
@@ -761,6 +814,8 @@
                         Background="{TemplateBinding Background}"
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
+                                  Margin="5,2,5,2"
```

Comment:
nit: fix indentation

Reply:
nit: shorten to 5,2

---
In reply to: [556748138](https://github.com/microsoft/microsoft-ui-xaml/pull/3938#discussion_r556748138) [](ancestors = 556748138)